src/Preferences.cpp

```diff
@@ -173,7 +173,7 @@ size_t Preferences::putString(const char* key, const String value){
 }
 
 size_t Preferences::putBytes(const char* key, const void* buf, size_t len){
-    if(!_started || !key || !buf || !len || _readOnly){
+    if(!_started || !key || !buf || len < 0 || _readOnly){
```

Comment:
size_t can't be a negative numbers, so this len check should be removed completely in that case

Reply:
you're right
If in the future there is interest in this PR at all, I'll update it  for merging ;-)